e2e/test_0000_setup_verify.py

```diff
@@ -0,0 +1,112 @@
+from os import path
+
+import pytest
+from kubernetes import client, config, utils
+
+
+@pytest.fixture
+def core_v1_client():
+    config.load_kube_config()
+    return client.CoreV1Api()
+
+
+def gluster_pods_verify(pods):
```

Comment:
expected pod count needs to be the input

Reply:
we can have a generic function which takes `namespace` `pod name` and `expected count` as input.

Reply:
good idea. ack